### PR TITLE
fix(docs): update custom cache TTL

### PR DESCRIPTION
**Pull request by Athira001**
Docs- update custom cache TTL

### CODE REVIEW

File: tutorial/custom-cache-ttl.md [hunk 1]
```diff
@@ -7,12 +7,36 @@ nav_exclude: true
 
 # {{page.title}}
 
-_This tutorial was contributed by [Amogh](https://github.com/ags1773)_
+_This tutorial was contributed by [Amogh](https://github.com/ags1773)_ and [Athira](https://www.linkedin.com/in/athira-m-r-835ab6105/)
```
Comment (cricket-lover) [inline tutorial/custom-cache-ttl.md:10]: can we remove underscore before "and" and put it at the end of the the sentence(to make the entire text italic) 
```suggestion
_This tutorial was contributed by [Amogh](https://github.com/ags1773) and [Athira](https://www.linkedin.com/in/athira-m-r-835ab6105/)_
```

File: tutorial/custom-cache-ttl.md [hunk 2]
```diff
@@ -7,12 +7,36 @@ nav_exclude: true
...
-- Make sure [@quintype/framework](https://www.npmjs.com/package/@quintype/framework) version is `4.16.2` or higher
+#### How to override  s-maxage value in Isomorphic Routes
 
+For overriding the s-maxage value, We can set `isomorphicRoutesSmaxage: <value>` under publisher in publisher.yml config file that comes from BlackKnight or pass it from the app level.  The default value is set to 900 seconds.
+
+- Make sure [@quintype/framework](https://www.npmjs.com/package/@quintype/framework) version is `5.0.5` or higher
+
+**How to pass it from Black Knight **
+
+Go to Black Knight  `/app/config/publisher.yml`,  add `isomorphicRoutesSmaxage:  <value>` under `publisher`.
```
Comment (shraddha-kesari) [inline tutorial/custom-cache-ttl.md:22]: Remove space from Black Knight. And add this URL https://black-knight.quintype.com/

File: tutorial/custom-cache-ttl.md [hunk 3]
```diff
@@ -7,12 +7,36 @@ nav_exclude: true
 
 # {{page.title}}
 
-_This tutorial was contributed by [Amogh](https://github.com/ags1773)_
+_This tutorial was contributed by [Amogh](https://github.com/ags1773)_ and [Athira](https://www.linkedin.com/in/athira-m-r-835ab6105/) 
 
-The time for which resources are cached on CDN i.e. the s-maxage value in the Cache-Control header is set to 900 seconds for most requests. This can be changed if needed from the app level.
+TTL is nothing but the time for which resources are cached on CDN. The value `s-maxage` is stored in a part of the response called the HTTP header, and it specifies for how many seconds, minutes, or hours content will be cached. In this tutorial, we are going to discuss how to override  `s-maxage` value in response cache-control header for `isomorphicRoutes` and `upstreamQuintypeRoutes`.
```
Comment (shraddha-kesari) [inline tutorial/custom-cache-ttl.md:12]: Remove nothing but

File: tutorial/custom-cache-ttl.md [hunk 4]
```diff
@@ -21,10 +45,44 @@ The time for which resources are cached on CDN i.e. the s-maxage value in the Ca
...
+
+- However, it won't change the s-maxage value of some requests like service-worker, shell.html, app manifest
+
+#### How to override  s-maxage value in Upstream Quintype Routes
+
+For overriding the s-maxage value for upstream routes(sketches routes),  set `upstreamRoutesSmaxage:  <value>` under publisher in publisher.yml config file that comes from BlackKnight or pass it from the app level. By default, the s-maxage value  will be the same as how it's set in sketches.
+
+- Make sure [@quintype/framework](https://www.npmjs.com/package/@quintype/framework) version is `5.0.5` or higher
+
+How to pass it from Black Knight
```
Comment (shraddha-kesari) [inline tutorial/custom-cache-ttl.md:62]: Make it as a heading

File: tutorial/custom-cache-ttl.md (hunk 1 above)
Comment (Athira001) [reply tutorial/custom-cache-ttl.md:10]: done

File: tutorial/custom-cache-ttl.md (hunk 2 above)
Comment (Athira001) [reply tutorial/custom-cache-ttl.md:22]: done

File: tutorial/custom-cache-ttl.md (hunk 4 above)
Comment (Athira001) [reply tutorial/custom-cache-ttl.md:62]: done

File: tutorial/custom-cache-ttl.md (hunk 3 above)
Comment (Athira001) [reply tutorial/custom-cache-ttl.md:12]: done

**Review by cricket-lover — APPROVED** [review]

File: tutorial/custom-cache-ttl.md [hunk 5]
```diff
@@ -21,10 +45,44 @@ The time for which resources are cached on CDN i.e. the s-maxage value in the Ca
...
+**How to pass it from app level**
+- In the `server/app.js` [file](https://github.com/quintype/malibu/blob/master/app/server/app.js), pass an option `sMaxAge: <value>` to the `upstreamQuintypeRoutes` function like so:
+
+  ```js
+  upstreamQuintypeRoutes(app, {sMaxAge: 1800});
+
+  ```
 
-- However it won't change the s-maxage value of some requests like service-worker, shell.html, app manifest
+- For `Breaking News(api/v1/breaking-news), /qlitics.js` and if the `cacheability is Private`, it is not overridden, instead the cache control will be the same as how it's set in sketches.
```
Comment (rashmirmittal) [inline tutorial/custom-cache-ttl.md:88]: Add header - Exceptions for s-maxage

Comment (Athira001) [reply tutorial/custom-cache-ttl.md]: Done

**Pull request merged**